bots/acl_delete.py

```diff
@@ -0,0 +1,53 @@
+'''
+## acl_delete
+What it does: deletes created network acl.
+Usage: AUTO: acl_delete
+
+Sample GSL: cloudtrail where event.name='CreateNetworkAcl'
+Limitation: Bot will not delete default vpc's network acl
+Note: Logic only bot

```

Comment:
should this note be here? 
Is compliance alert provides acl id in entity when calling the bot? 
if not the note is relevant.
@RavidAtias

Reply:
Is there an option, ACL is returned as an entity in Compliance? @RavidAtias